ceph/UI/dashboard.py

```diff
@@ -0,0 +1,66 @@
+from selenium.common.exceptions import TimeoutException, WebDriverException
+from selenium.webdriver.common.by import By
+from selenium.webdriver.support import expected_conditions as EC
+from selenium.webdriver.support.ui import WebDriverWait
+
+
+class Dashboard:
+    def __init__(self, driver):
+        """
```

Comment:
Please get the driver initilialised from this class and return the driver from browser.py.

Reply:
I have updated the code accordingly. Kindly have a look at it and suggest if any further modifications are required. Thanks.